test/Vagrantfile

```diff
@@ -40,7 +40,7 @@ $CILIUM_REGISTRY = ENV['CILIUM_REGISTRY'] || ""
 $MEMORY = (ENV['VM_MEMORY'] || "4096").to_i
 $CPU = (ENV['VM_CPUS'] || "2").to_i
 
-if ENV['NETNEXT'] == "true" || ENV['NETNEXT'] == "1" || ENV['KERNEL'] == "net-next" then
```

Comment:
It would be nice if the environment variables for the dev VM and test VM are same. I personally would prefer to continue to use `NETNEXT` for the dev VM (I already have it stored in my IDE settings/scripts), and hence, test VM as well.

Reply:
I think the goal is to try and use a single env. variable for everything. It's a bit confusing to see both `NETNEXT` and `KERNEL`. Clearly it needs to be `KERNEL` (imagine `NETNEXT=419` :laughing:).

I'm happy if we can move to just `KERNEL` for both the test and the development VM. Not sure if we can do that without first deprecating `NETNEXT` for the development VM; maybe a message in #development would be enough :thinking:

Reply:
So I've gone the most “extreme” path, and updated the variables both for CI and development setups. The new version uses `KERNEL=netnext` everywhere, `NETNEXT` variants are all removed (Sorry Aditi, I agree with Paul on `KERNEL` being more consistent). I'll link the PR on Slack so we can gather more feedback.